pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -423,40 +416,45 @@ def process_circuits(
                 if self._MACHINE_DEBUG:
                     for i, ind in enumerate(indices_chunk):
                         handle_list[ind] = ResultHandle(
-                            _DEBUG_HANDLE_PREFIX
-                            + str((batch_chunk[i].n_qubits, n_shots, batch_id)),
+                            _DEBUG_HANDLE_PREFIX + str((n_shots, batch_id)),
                             i,
+                            batch_chunk[i].n_qubits,
                             ppcirc_strs[i],
                         )
                 else:
-                    job = self._backend.run(
-                        qcs, shots=n_shots, memory=self._config.memory
-                    )
-                    jobid = job.job_id()
+                    options = Options()
+                    options.resilience_level = 0
+                    options.execution.shots = n_shots
```

Comment:
Looks like the the [Options](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/options/options.py) have default settings:
```
options.optimization_level = 3
options.transpilation.skip_transpilation = False
```

I'm not sure how this will affect our compilations in terms of benchmarking and error mitigation. I also don't know whether the transpilation setting will ignore the tket qubit placement.

Reply:
Yes that's true. I set `resilience_level` to 0 to turn off error mitigation; I should probably also set `transpilation.skip_transpilation` to `True` to preserve TKET placement, and even `optimization_level` to 0 so that we have control over gates. In a follow-up PR we can make all these configurable.